nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const app = express();
+// Parse URL-encoded bodies (as sent by HTML forms)
+app.use(express.urlencoded({ extended: true }));
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json());
+app.get("/calculator", (req, res) => {
```

Comment:
Most of your logic in addition/subtraction is correct, however it's not exactly what the [hw](https://github.com/HackYourFuture-CPH/node.js/tree/main/week3/homework-template#getting-data-through-query-parameters-using-get) wanted you to do:

- While what you did works, it is too complicated to include everything in a single route. You should have created 4 separate routes for the addition/subtraction/multiplication/division. Eg:
```
app.get("/calculator/addition", (req, res)
```
- You're also missing the [post requests](https://github.com/HackYourFuture-CPH/node.js/tree/main/week3/homework-template#getting-data-through-the-request-body-using-post). What you have implemented is a get request, meaning you will get the parameters from the url and the req.query. You also have to implement a post request and in that case you will get the parameters from the request's body and the req.body instead of req.query

Reply:
Thanks for the comments. in hw we do not had to separate the add,sub,mul,and divide. but i know how to do it. this code is giving the needed result and i put it post part aswell. thanks.